e2e/test/scenarios/admin/admin-reproductions.cy.spec.js

```diff
@@ -54,3 +63,68 @@ describe("issue 21532", () => {
     });
   });
 });
+
+describe("issue 41765", { tags: ["@external"] }, () => {
+  // In this test we are testing the in-browser cache that metabase uses,
+  // so we need to navigate by clicking trough the UI without reloading the page.
+
+  const WRITABLE_DB_DISPLAY_NAME = "Writable Postgres12";
+
+  const TEST_TABLE = "scoreboard_actions";
+  const TEST_TABLE_DISPLAY_NAME = "Scoreboard Actions";
+
+  const COLUMN_NAME = "another_column";
+  const COLUMN_DISPLAY_NAME = "Another Column";
+
+  beforeEach(() => {
+    resetTestTable({ type: "postgres", table: TEST_TABLE });
+    restore("postgres-writable");
+    cy.signInAsAdmin();
+
+    resyncDatabase({
+      dbId: WRITABLE_DB_ID,
+      tableName: TEST_TABLE,
+    });
```

Comment:
Redundant

```suggestion
```

Reply:
The test does not work for me locally if I remove this call.